lib/picam/plug_streamer.ex

```diff
@@ -0,0 +1,39 @@
+if Code.ensure_loaded?(Plug) do
```

Comment:
I've been thinking about this if statement a lot. I'm guessing that you evaluated 3 options:
1. Depend on `plug` unconditionally
2. Make a second package - call it `picam_plug`
3. Make `plug` optional, but if it's around, so is `Picam.PlugStreamer`
I'm having trouble coming to terms with option 3. This may be due to the side effecty nature of optional dependencies that bugs me or it could be that everything else in the file needs to be tabbed out 2 more spaces and that looks weird.

Could I convince you to pick option 1 or 2 instead?

Also, not sure if this is coming, but I think that one of us needs to add documentation to this.

Reply:
My vote is for option 2.  You could then also include the router in the new package.

Reply:
@mobileoverlord If you would rather I create and maintain that package, let me know.